Sources/DocCDocumentation/DocCDocumentation.docc/RevenueCat.md

```diff
@@ -159,7 +159,7 @@ Or browse our iOS sample apps:
 - ``Purchases/setOnesignalID(_:)``
 
 ### Configuring the SDK with parameters (deprecated)
-- ``Purchases/configure(withAPIKey:appUserID:)``
-- ``Purchases/configure(withAPIKey:appUserID:observerMode:storeKitVersion:)``
+- ``Purchases/configure(withAPIKey:appUserID:)-57pv0``
+- ``Purchases/configure(withAPIKey:appUserID:purchasesAreCompletedBy:storeKitVersion:)-2k6md``
```

Comment:
Do you know where these suffixes are coming from?

Reply:
Yes, if the method it's referencing is overloaded, then it uses these suffixes to distingish between them. Xcode provides the suffixes via a "fix":

<img width="1076" alt="image" src="https://github.com/RevenueCat/purchases-ios/assets/109382862/b0c43540-7541-4896-9901-081a1005dece">

Reply:
Oh interesting